dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -87,11 +87,16 @@ RUN pip install allure-pytest==2.8.22 \
                 rpyc \
                 six \
                 snappi[ixnetwork,convergence]==0.7.44 \
+                ixload \
                 statistics \
                 tabulate \
                 textfsm==1.1.3 \
                 thrift==0.11.0 \
                 virtualenv \
+                dpugen==0.0.5 \
+                ctypesgen \
+                pandas \
+                PyYAML \
```

Comment:
Can you add the dependent packages to python3 environment too? We are in the progress of migrating to python3 and deprecating python2 environment.
BTW, can you add the packages in alphabetic order?

Reply:
Added for python3 also